recipes/opencv/4.x/conanfile.py

```diff
@@ -31,7 +31,8 @@ class OpenCVConan(ConanFile):
         "with_gtk": [True, False],
         "with_quirc": [True, False],
         "with_cuda": [True, False],
-        "with_cublas": [True, False]
+        "with_cublas": [True, False],
+        "with_protobuf": [True, False]
```

Comment:
Just a general comment for CCI: do we really want to mimic upstream options names, instead of translating to something more meaningful.
I mean, when I see `with_protobuf`, I have no idea of the provided feature, which is the most important thing.

Reply:
I can use "with_dnn", that would be more meaningful

Reply:
or just `dnn`, `with_` is useless

Reply:
We absolutely need to document this, I saw you replied but https://github.com/conan-io/conan-center-index/pull/5286#discussion_r618833025

Implies this should be an "enable" since it turns on a feature (and adds a dependency)